tests/gui/measure_script/test_script_edit_dialog.py

```diff
@@ -97,6 +97,18 @@ def test_try_save(
                     errmsg_mock.assert_called_once()
 
 
+def test_save_load_roundtrip(dlg: ScriptEditDialog, tmp_path: Path) -> None:
```

Comment:
It doesn't look like this is causing issues currently, but I think there should be `qtbot` in the arguments as you are making a `QWidget`:

```suggestion
def test_save_load_roundtrip(dlg: ScriptEditDialog, tmp_path: Path, qtbot: QtBot) -> None:
```

(The qtbot just sets up a fake Qt application context for the test. Without it you sometimes get segfaults if you try to use Qt functionality.)